src/components/EventSubMenu.js

```diff
@@ -26,23 +26,56 @@ const eventSubMenu = [
     path: '/messages',
   },
 ];
-export default function EventSubMenu() {
+export default function EventSubMenu(props) {
+  const { name, desc, start, end, timezone, id, googleEventId } = props.event;
+  const { dispatch } = props;
+  const { edit, setEdit } = props.edit;
+
+  const publishTheCalendarEvent = (event) => {
+    if (!googleEventId) {
+      try {
+        window.gapi.client.load('calendar', 'v3', () => {
+          var request = window.gapi.client.calendar.events.insert({
+            calendarId: 'primary',
+            resource: event,
+          });
+
+          request.execute((event) => {
+            console.log('Event created: ', event);
+            window.open(event.htmlLink);
+            dispatch({
+              type: 'addGoogleEventId',
+              eventId: id,
+              googleEventId: event.id,
+            });
+          });
+        });
+      } catch (error) {
+        console.log(error);
+      }
+    } else console.log('event already created');
```

Comment:
These console.logs could be `console.error`. Or the second one maybe even a `console.warn`.

Reply:
Fixed with 9ccc540